tests/test_astrometryTask.py

```diff
@@ -162,6 +164,17 @@ def doTest(self, pixelsToTanPixels, order=3):
         )
         self.assertLess(len(resultsRefSelect.matches), len(results.matches))
 
+        # try again, allowing magnitude outlier rejection.
```

Comment:
If you're going to keep this test--and given your new one, I don't know what point this serves--you should at least mention why you expect the same number of matches. If you could tell that setting the config actually changed the result, it might be worth keeping, but the test doesn't even tell us if the code was run (outside of looking at the coverage plot).

I think I'd advocate for restoring the code you added that made a few things outliers, and now you don't have to be careful about what kind of outlier, since all you care about is "did the outlier rejection actually run".

Reply:
I wanted to leave these tests as "null" tests to just show that the code does no harm if everything is perfect.  However, it's true that there's no other way to know that the code was actually run.  So I'll add in the bad guys.